x-pack/legacy/plugins/apm/public/components/shared/MetadataTable/SpanMetadata/__test__/SpanMetadata.test.tsx

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import 'jest-dom/extend-expect';
+import { render, cleanup } from 'react-testing-library';
+import { SpanMetadata } from '..';
+import { Span } from '../../../../../../typings/es_schemas/ui/Span';
+import { expectTextsInDocument } from '../../../../../utils/testHelpers';
+
+describe('SpanMetadata', () => {
+  afterEach(cleanup);
+  it('should render', () => {
```

Comment:
This is just my personal preference but I would break down the cases like:

```
describe('SpanMetadata', () => {
  describe('render', () => {
    it('renders', () => {...})

    describe('when a span is present', () => {
      it('renders the span', () => { ... })
    })
  })
})
```

While it's written for RSpec, I tend to try to follow the guidelines on http://www.betterspecs.org/.